docs/content/preview/yugabyte-platform/manage-deployments/xcluster-replication/xcluster-replication-setup.md

```diff
@@ -0,0 +1,272 @@
+
+- They have network connectivity; see [Networking for xCluster](../../../prepare/networking/#networking-for-xcluster). If the source and target universe Master and TServer nodes use DNS addresses, those addresses must be resolvable on all nodes.
+
+Prepare your database and tables on the source. The source can be empty or have data. If the source has a lot of data, setup will take longer because the [data must be copied in full to the target](#full-copy-during-xcluster-setup) before on-going asynchronous replication starts.
+
+During initial setup, you don't need to create objects on the target. YugabyteDB Anywhere performs a full copy of the data to be replicated on the source, and automatically creates tables and objects, and restores data on the target from the source.
+
+## Best practices
+
+- Monitor CPU use and keep it below 65%.
+- Monitor disk space and keep its use under 65%.
+- Set the YB-TServer [log_min_seconds_to_retain](../../../../reference/configuration/yb-tserver/#log-min-seconds-to-retain) flag to 86400 on both source and target.
+
+    This flag determines the duration for which WAL is retained on the source in case of a network partition or a complete outage of the target. For xCluster replication, you should set the flag to a value greater than the default. The goal is to retain write-ahead logs (WALs) during a network partition or target outage until replication can be restarted. Setting this to 86400 (24 hours) is a good rule of thumb, but you should also consider how quickly you will be able to recover from a network partition or target outage.
+
+    In addition, during an outage, you will need enough disk space to retain the WALs, so determine the data change rate for your workload, and size your disk accordingly.
+
+- [Set a replication lag alert](#set-up-replication-lag-alerts) for the source to be alerted when the replication lag exceeds acceptable levels.
+- Add new tables and databases to the replication configuration soon after creating them, and before performing any writes to avoid the overhead of a full copy.
+- If xCluster replication setup clashes with scheduled backups, wait for the scheduled backup to finish, and then restart the replication.
+
+## Configure replication
+
+### Before you start
+
+Before configuring replication, you should be aware of the following:
+
+- When a full copy of data needs to be made
+- Special conditions for adding YSQL tables to replication
+
+#### Full copy during xCluster setup
```

Comment:
This section and YSQL tables below could be potentially condensed or moved to the end as it currently feels like a long intro before we get to the main setup section.